examples/analog_in/voltage_acq_int_clk_plot_data.py

```diff
@@ -0,0 +1,20 @@
+﻿"""Example of generating visualizations for acquired data.
+
+This example demonstrates how to plot the acquired data.
```

Comment:
I would put a blurb mentioning that this requires pip installing matplotlib.

Reply:
Ooh, better yet, you could try/catch the import and print a message if it fails!

Reply:
We will add matplotlib and nptdms packages as dependencies for examples, do we still need to mention the `pip install matplotlib` in this example? And the try/catch for importing the library, I think maybe all these things are not needed anymore, right?

Reply:
Yeah, I think the try/catch is a good idea. I expect people will just copy/paste the examples and try to run them.

Reply:
I think a comment is sufficient. If you add a try/catch, people are likely to copy/paste it into their code.

Reply:
That's fair. A comment is fine by me.

Reply:
It seems like matplotlib is not supported on python 32-bit, the CI win-10-py32 shows 

```
PoetryException

  Failed to install C:/Users/ec2-user/AppData/Local/pypoetry/Cache/artifacts/18/be/b2/c68f3e7f22c04f84a1e658deab0e526a2f71c8a221cb585f3616afd348/matplotlib-3.9.0.tar.gz
```

I get the similar error after I switch to python 32-bit on my test machine when I execute poetry install.

```
- Installing matplotlib (3.9.0): Failed

  ChefBuildError

  Backend subprocess exited when trying to invoke build_wheel
  
  + meson setup C:\Users\jocheah\AppData\Local\Temp\tmp_fxi70na\matplotlib-3.9.0 C:\Users\jocheah\AppData\Local\Temp\tmp_fxi70na\matplotlib-3.9.0\.mesonpy-_r8xcjka -Dbuildtype=release -Db_ndebug=if-release -Db_vscrt=md --native-file=C:\Users\jocheah\AppData\Local\Temp\tmp_fxi70na\matplotlib-3.9.0\.mesonpy-_r8xcjka\meson-python-native-file.ini
  The Meson build system
  Version: 1.4.1
  Source dir: C:\Users\jocheah\AppData\Local\Temp\tmp_fxi70na\matplotlib-3.9.0
  Build dir: C:\Users\jocheah\AppData\Local\Temp\tmp_fxi70na\matplotlib-3.9.0\.mesonpy-_r8xcjka
  Build type: native build
  Program python3 found: YES
  Project name: matplotlib
  Project version: 3.9.0
  Activating VS 17.8.4
  C compiler for the host machine: cl (msvc 19.38.33134 "Microsoft (R) C/C++ Optimizing Compiler Version 19.38.33134 for x64")
  C linker for the host machine: link link 14.38.33134.0
  C++ compiler for the host machine: cl (msvc 19.38.33134 "Microsoft (R) C/C++ Optimizing Compiler Version 19.38.33134 for x64")
  C++ linker for the host machine: link link 14.38.33134.0
  Host machine cpu family: x86_64
  Host machine cpu: x86_64
  Program python found: YES (C:\Users\jocheah\AppData\Local\Temp\tmp17r72o4i\.venv\Scripts\python.exe)
  Need python for x86_64, but found x86
  Run-time dependency python found: NO (tried sysconfig)
```

I can't find a way to specify the matplotlib package is only installed for python 64-bit in pyproject.toml file. I think I have to remove matplotlib from pyproject.toml file, and then skip the example from pytest. In the docstring, remind users to pip install matplotlib in order to run this example.

Reply:
I found the matplotlib commit that stopped building Windows 32-bit wheels: https://github.com/matplotlib/matplotlib/commit/da5aaeb1e2d6f98bbbb2416ed102c9e9bc361da7

I asked @mshafer-NI and we couldn't come up with a way to specify this either, since https://peps.python.org/pep-0508/#environment-markers doesn't distinguish between 32-bit and 64-bit Windows.

How about making the `example` group optional? You would run `poetry install --with example` to install it. It probably makes sense to rename it to `examples`.
```python
[tool.poetry.group.examples]
# The "example" group is optional because matplotlib does not distribute wheels for 32-bit Windows.
optional = true

[tool.poetry.group.examples.dependencies]
matplotlib = {version=">=3.9.0", python = ">=3.9"}
nptdms = ">=1.9.0"
```